src/ORM/Table.php

```diff
@@ -1677,10 +1677,19 @@ public function findOrCreate($search, callable $callback = null, $options = [])
     {
         $options += [
             'atomic' => true,
-            'defaults' => true
+            'defaults' => true,
+            '_primary' => true,
```

Comment:
New option should go in the doc block too.

Reply:
Turns out this isn't necessary for this method, since it's always primary. I was going to document it on `save()` since it's undocumented, but it seems like an internal option.